src/components/HOCs/WithSearchRoute/WithSearchRoute.test.js

```diff
@@ -20,7 +20,8 @@ test("executeRouteSearch clears old redux values first", () => {
   const WrappedComponent = WithSearchRoute(() => <div className="child" />, "challenges")
   basicProps.history = {location: {search: "?query=test"}}
 
-  const wrapper = shallow(
+  // const wrapper = shallow(
+  shallow(
```

Comment:
I think these lines can just be deleted; the comment isn't really adding anything helpful. (Same goes for the other instances in this file)